index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+<body>
+<div class="all">
+    <nav>
+        <a href=""><img src="./images/My Logo.png" alt="logo holder"></a>
+        <a href=""><img src="./images/Union.png" alt="humbergar"></a>
```

Comment:
- [ ] [OPTIONAL] Great job using a `nav` tag 💪 .  Kindly use a `header` tag to wrap all the elements in the `nav` tag 🙏 
Example:
```
<header>
  <nav>
  </nav>
</header>
```